admin/pages/help-settings.php

```diff
@@ -17,14 +17,21 @@
 
 $wi_form = new WI_Volunteer_Management_Form();
 $wi_form->admin_header();
+
+// Filter to allow the Help & Settings menu 'Help' tab to be hidden
+$wivm_show_help_tab	= apply_filters( 'wivm_show_help_tab', true );
```

Comment:
Nice work on this. It serves the purpose well and I could see others using it also.